src/hot-keys/hotkeyStore.ts

```diff
@@ -128,13 +128,13 @@ const mod = {
         hotkeys.forEach(h => {
           h.units.forEach(u => {
             u.abilities.forEach(a => {
-              if (a.hotkeyIdentifier === hk.hotkeyCommand) {
+              if (a.hotkeyIdentifier === hk.hotkeyCommand || a.additionalHotkeyIdentifiers.some(hki => hki === hk.hotkeyCommand)) {
```

Comment:
added this so that it includes the extra hotkey identifiers like "Upgrade to Castle/Black Citadel", etc, which were being excluded